<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::MainWindow</class>
 <widget class="QMainWindow" name="GammaRay::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="SidePane" name="toolSelector">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="selectionRectVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="toolStack">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>39</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_GammaRay">
    <property name="title">
     <string>&amp;GammaRay</string>
    </property>
    <addaction name="actionRetractProbe"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <widget class="QMenu" name="menu_Diagnostics">
     <property name="title">
      <string>&amp;Diagnostics</string>
     </property>
     <addaction name="actionPlugins"/>
     <addaction name="actionMessageStatistics"/>
    </widget>
    <addaction name="actionHelp"/>
    <addaction name="actionContribute"/>
    <addaction name="separator"/>
    <addaction name="menu_Diagnostics"/>
    <addaction name="separator"/>
    <addaction name="actionAboutGammaRay"/>
    <addaction name="actionAboutKDAB"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="actionsMenu">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Setti&amp;ngs</string>
    </property>
   </widget>
   <addaction name="menu_GammaRay"/>
   <addaction name="actionsMenu"/>
   <addaction name="menuSettings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionRetractProbe">
   <property name="text">
    <string>&amp;Detach</string>
   </property>
   <property name="toolTip">
    <string>Retract probe from application.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit probed application.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAboutGammaRay">
   <property name="icon">
    <iconset resource="../resources/gammaray.qrc">
     <normaloff>:/gammaray/GammaRay-32x32.png</normaloff>:/gammaray/GammaRay-32x32.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;GammaRay</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionAboutKDAB">
   <property name="text">
    <string>About &amp;KDAB</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPlugins">
   <property name="text">
    <string>&amp;Plugins...</string>
   </property>
  </action>
  <action name="actionMessageStatistics">
   <property name="text">
    <string>&amp;Message Statistics...</string>
   </property>
   <property name="toolTip">
    <string>Show GammaRay communication statistics.</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help...</string>
   </property>
   <property name="toolTip">
    <string>Show help browser.</string>
   </property>
  </action>
  <action name="actionContribute">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Contribute...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SidePane</class>
   <extends>QListView</extends>
   <header>sidepane.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/gammaray.qrc"/>
 </resources>
 <connections/>
</ui>
